homeassistant/components/sensor/mqtt.py

```diff
@@ -28,6 +32,7 @@
 PLATFORM_SCHEMA = mqtt.MQTT_RO_PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_UNIT_OF_MEASUREMENT): cv.string,
+    vol.Optional(CONF_EXPIRE_AFTER, default=0): cv.positive_int,
```

Comment:
Can we remove the default and just check `if self._expire_after is not None:` below?